src/NUnitEngine/nunit.engine/Runners/ProcessRunner.cs

```diff
@@ -261,7 +261,7 @@ protected override void Dispose(bool disposing)
                 }
 
                 if (unloadError != null) // Add message line indicating we managed to stop agent anyway
-                    throw (new NUnitEngineException(unloadError + "\nAgent Process was terminated successfully after error."));
+                    throw (new NUnitEngineException(unloadError + Environment.NewLine + "Agent Process was terminated successfully after error."));
```

Comment:
Perhaps we should also remove the parentheses surrounding `new NUnitEngineException(....)`